misc/windows-iam/Setup_Iam.ps1

```diff
@@ -11,10 +11,40 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-Invoke-Expression "${PSScriptRoot}\loopback.ps1"
-Invoke-Expression "${PSScriptRoot}\hostsetup.ps1"
-Invoke-Expression "docker build -t amazon/amazon-ecs-credential-proxy --file ${PSScriptRoot}\credentialproxy.dockerfile ${PSScriptRoot}"
+$ErrorActionPreference = 'Stop'
 
+# Download devcon binary
+$devconZipUri = "https://s3.amazonaws.com/amazon-ecs-devcon/devcon-3a3f98688359aa1b905147831f910218.zip"
+$devconZipMD5Uri = "$devconZipUri.md5"
+$devconZipFile = "${PSScriptRoot}\devcon.zip"
```

Comment:
> This suggests that for the purpose of tests, we are copying devcon.exe to the windows-iam folder.

Yes, that's correct.

> is it packaged as part of the ecs agent zip ?

Yes, see [here](https://github.com/samuelkarp/amazon-ecs-agent/blob/dc3924e4835c9e113beadc144c02a1559d7d59db/scripts/stage#L87-L101).

> Other than this, if the below re-organization has been tested, then it looks good to go.

Yep, it's passing for me.